db/migrate/20180522142853_create_zipcode_rates.rb

```diff
@@ -0,0 +1,14 @@
+class CreateZipcodeRates < ActiveRecord::Migration
+  def change
+    create_table :zipcode_rates do |t|
+      t.string :zip_code
```

Comment:
Does this need a `version` in order to track entries based on the source file that was uploaded?

Reply:
I do not think for this model we need version.

Reply:
FYI I may be misguided on ^.

Reply:
I don't think we will need version for this model